recipes/wcurl/bld.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+
+REM Create the bin directory if it doesn't exist
+if not exist "%PREFIX%\bin" (
+    mkdir "%PREFIX%\bin"
+)
+
+REM Copy the file and check for success
+copy "%SRC_DIR%\wcurl" "%PREFIX%\bin\wcurl" >nul
+if errorlevel 1 (
+    echo ERROR: Failed to copy wcurl to %PREFIX%\bin.
+    exit /b 1
+)
```

Comment:
Think we either want `%LIBRARY_BIN%` or `%SCRIPTS%` as those would be on `%PATH%`

ref: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process

Reply:
It seems that the `wcurl` command is not being found. Do I need to make anything else to make it available?
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1105776&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de&l=1763